CMakeLists.txt

```diff
@@ -37,7 +37,7 @@ endif()
 #-----------------------------------------------------------------------------
 add_subdirectory(VesselnessFiltering)
 add_subdirectory(LevelSetSegmentation)
-if(${Slicer_VERSION_MAJOR}.${Slicer_VERSION_MINOR} VERSION_GREATER_EQUAL 4.11)
+if(${Slicer_VERSION_MAJOR}.${Slicer_VERSION_MINOR} VERSION_GREATER_EQUAL 4.11 OR ${Slicer_VERSION_MAJOR} EQUAL 0)
```

Comment:
I just tried to add the statement `message("Slicer_VERSION_MINOR:${Slicer_VERSION_MINOR}")` into the CMakeLists.txt of a custom app module I generated yesterday, and it displays the appropriate value.

Out of curiosity, how is the module integrated ?

Reply:
In the main CMakeLists.txt of the custom app we have

```
# SlicerVMTK
set(extension_name "SlicerVMTK")
set(${extension_name}_SOURCE_DIR "${CMAKE_BINARY_DIR}/${extension_name}")
FetchContent_Populate(${extension_name}
 SOURCE_DIR     ${${extension_name}_SOURCE_DIR}
 GIT_REPOSITORY ${EP_GIT_PROTOCOL}://github.com/vmtk/SlicerExtension-VMTK.git
 GIT_TAG        3787ea4a300da28ec5f0824f0715f2713b631155
 GIT_PROGRESS   1
 QUIET
 )
list(APPEND Slicer_EXTENSION_SOURCE_DIRS ${${extension_name}_SOURCE_DIR})
```